packages/venia-ui/__mocks__/react-intl.js

```diff
@@ -7,6 +8,8 @@ const intl = reactIntl.createIntl({
 
 module.exports = {
     ...reactIntl,
-    FormattedMessage: jest.fn(({ defaultMessage, id }) => defaultMessage || id),
+    FormattedMessage: props => (
+        <div componentName="Formatted Message Component" {...props} />
```

Comment:
Jest suggests mock components using this convention, which I think I'd prefer so we're not adding semantic markup.

https://jestjs.io/docs/en/tutorial-react#snapshot-testing-with-mocks-enzyme-and-react-16

```suggestion
        <mock-FormattedMessage>" {...props} />
```